src/models/deck.model.ts

```diff
@@ -30,9 +30,8 @@ class Deck {
 
 export function numberOfCards(card: Card, deckCards: Card[]) {
```

Comment:
This functionality could be created without creating this function by using the filter method.
An example would be as follows
```
while (deckCards.filter(c => card.name === card.name).length !== 5) { deckCards.push(card); }
```